src/net_processing.cpp

```diff
@@ -3114,8 +3118,33 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
     }
 
     if (strCommand == NetMsgType::NOTFOUND) {
-        // We do not care about the NOTFOUND message, but logging an Unknown Command
-        // message would be undesirable as we transmit it ourselves.
+        // NOTFOUND means we can clear the entry from the tx_in_flight/announced set
+        std::vector<CInv> vInv;
+        vRecv >> vInv;
+        //if (vInv.size() > MAX_INV_SZ)
+        //{
+        //    LOCK(cs_main);
+        //    Misbehaving(pfrom->GetId(), 20, strprintf("message notfound size() = %u", vInv.size()));
+        //    return false;
+        //}
```

Comment:
In commit 392cf539756e2af6df5684c3444df626b446754f

What is this adding? We generally don't add commented out code without rationale

Reply:
Replaced with actual comment